Source/Urho3D/Urho2D/TileMapDefs2D.h

```diff
@@ -126,7 +133,13 @@ class URHO3D_API Tile2D : public RefCounted
     Tile2D();
 
     /// Return gid.
-    int GetGid() const { return gid_; }
+    int GetGid() const { return gid_ & ~FLIP_ALL; }
```

Comment:
May be should change to unsigned here also?

Reply:
Here, there, and everywhere, got it!